src/core.c/Main.pm6

```diff
@@ -276,7 +276,32 @@ my sub RUN-MAIN(&main, $mainline, :$in-as-argsfiles) {
                     }
                     @positional.push($argument);
                 }
-                @arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
+            		#@arg-help.push($argument => $param.WHY.contents) if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty;  # Use first defined
+            		if $param.WHY and (@arg-help.grep:{ .key eq $argument}) == Empty {
```

Comment:
Is the indentation on this correct?

Reply:
Probably not, I will need to rework the PR with respect to indentation. I was more concerned with functionality when I submitted this PR. Expect another commit to correct the whitespacing.